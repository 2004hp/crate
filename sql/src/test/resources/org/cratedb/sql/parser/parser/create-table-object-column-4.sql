CREATE TABLE t4 ( title string, stuff object as (name string, age integer))