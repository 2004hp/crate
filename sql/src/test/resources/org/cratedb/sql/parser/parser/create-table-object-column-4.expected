org.cratedb.sql.parser.parser.CreateTableNode@3aa3a481
name: t4
statementType: CREATE TABLE
lockGranularity: R
existenceCheck: NO_CONDITION
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@4d852f3a

	[0]:	
	org.cratedb.sql.parser.parser.ColumnDefinitionNode@62dfb098
	type: string
	name: title
	elementType: null
	[1]:	
	org.cratedb.sql.parser.parser.ObjectColumnDefinitionNode@4bc2fdde
	type: OBJECT
	name: stuff
	elementType: null
	objectType: DYNAMIC
	subColumns: 		
		org.cratedb.sql.parser.parser.TableElementList@6ea920ad

		[0]:		
		org.cratedb.sql.parser.parser.ColumnDefinitionNode@21a0769d
		type: string
		name: name
		elementType: null
		[1]:		
		org.cratedb.sql.parser.parser.ColumnDefinitionNode@2dcb4eaa
		type: INTEGER
		name: age
		elementType: null
